.github/workflows/codecov.yaml

```
@@ -0,0 +1,31 @@
+        uses: actions/setup-node@v4
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Run tests
+        run: npx jest --coverage
+
+      - name: Upload results to Codecov
+        uses: codecov/codecov-action@v5
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
```

yamllint (1.35.1)

[warning] wrong indentation: expected 12 but found 10

(indentation)